nextcord/client.py

```diff
@@ -1873,6 +1872,37 @@ def add_view(self, view: View, *, message_id: Optional[int] = None) -> None:
 
         self._connection.store_view(view, message_id)
 
+    def remove_view(self, view: View) -> None:
+        """Removes a :class:`~nextcord.ui.View` from persistent listening.
+
+        This method should be used if a persistent view is set in a cog and
+        should be freed when the cog is unloaded to save memory.
+
+        .. versionadded:: 2.3
+
+        Parameters
+        ------------
```

Comment:
Unnecessary heading underlining.
```suggestion
        Parameters
        ----------
```